regulations/templates/regulations/preamble-partial.html

```diff
@@ -17,12 +17,11 @@
             <div class="queued"></div>
             <input type="file">
             <button type="submit">Save</button>
-            {# <div class="comment-clear">Clear this comment</div> #}
```

Comment:
So blanking a comment model doesn't destroy it--I inferred/guessed/imagined that @xtine commented out this line because we can now delete comments from the index on the side. I like interfaces that give the user exactly one way to perform a given action, so I think I prefer one delete widget (on the index) to two (one under the comment, another in the index). But now that you mention it, I didn't verify this. What do you think @xtine?